lib/driver.js

```diff
@@ -291,12 +291,6 @@ class XCUITestDriver extends BaseDriver {
     }
     this.logEvent('xcodeDetailsRetrieved');
 
-    if (this.opts.enableAsyncExecuteFromHttps && !this.isRealDevice()) {
-      // shutdown the simulator so that the ssl cert is recognized
-      await shutdownSimulator(this.opts.device);
-      await this.startHttpsAsyncServer();
```

Comment:
are `(start/stop)HttpsAsyncServer` used anywhere else? can these method also be removed?

Reply:
They are still used in `appium-ios-driver`, where the old method of doing things is still being used. The functions are pulled from there.